src/middlewares/auth.mdw.js

```diff
@@ -1,7 +1,10 @@
 export function isAuth(req, res, next) {
+  // console.log('Session Data:', req.session);
   if (!req.session.user) {
     return res.redirect('/login');
   }
+  req.user = req.session.user;
```

Comment:
I think this is unnecessary, pls explain

Reply:
Indeed, after thinking my task doesn't need to use isAurth function, I forgot to delete it.